spec/hydra/works/models/generic_work_spec.rb

```diff
@@ -13,15 +13,22 @@
 
   describe '#generic_works=' do
     it 'should aggregate generic_works' do
-      generic_work1.generic_works = [generic_work2, generic_work3]
-      expect(generic_work1.generic_works).to eq [generic_work2, generic_work3]
+      generic_work1.child_generic_works = [generic_work2, generic_work3]
+      expect(generic_work1.child_generic_works).to eq [generic_work2, generic_work3]
     end
   end
 
-  describe '#generic_files=' do
+  describe '#child_generic_files=' do
```

Comment:
Do we ever have parent_generic_files? I don't think we do, so I think making this distinction just extra clutter/typing. I'd prefer we just have "generic_files" We may need to discuss this on a call.